modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -459,10 +459,34 @@ This may not be specified in combination with `allowedDatabases`.
 
 As with the `DEALLOCATE DATABASES FROM SERVER ...` command, if the alteration of a server's options renders it impossible for the cluster to satisfy one or more of the databases' topologies, then the command fails and no changes are made.
 
-[NOTE]
+[IMPORTANT]
 ====
 Input provided to `SET OPTIONS {...}` replaces **all** existing options, rather than being combined with them.
-For instance if `SET OPTIONS {modeConstraint:'SECONDARY'}` is executed followed by `SET OPTIONS {allowedDatabases:['foo']}`, the execution of the second `ALTER` removes the mode constraint.
+
+Any previously set values must be specified every time you run the `ALTER SERVER` command; otherwise they will be overwritten to defaults taken from the settings `initial.server.mode_constraint`, `initial.server.allowed_databases`, `initial.server.denied_databases`, and/or `initial.server.tags`.
```

Comment:
This seems misleading. It sounds like if you don't specify some of the options, we will pick up the _configured setting_ from the relevant `initial.` setting you mention. In fact, we will just set the option to whatever the appropriate 'empty' value is.

Perhaps 
```suggestion
Any previously set values must be specified every time you run the `ALTER SERVER` command; otherwise they will be overwritten to the unset values.
```

The 'initial' values for a server are only relevant when you enable the server, if you don't specify something that overrides them then.

Reply:
ah, interesting. In the trello, we have, it's said: 

Similarly to the example given, if they (customers) have configuration for `allowedDatabases` or `deniedDatabases` and do not explicitly set them in the options for `ALTER SERVER`, the values are overwritten to defaults.

In the 'Enabled state' section, we say:
When a server is enabled, if no OPTIONS are not provided, the default server values are taken from the settings `initial.server.mode_constraint`, `initial.server.allowed_databases`, `initial.server.denied_databases`, and/or `initial.server.tags`.

That's why I decided that defaults are taken from the `initial.` settings.